### DISCUSSION TITLE
最強iPad！ iPad Pro 2022有望本月發：Apple M2加持

### DISCUSSION THREAD

**Discussion by williamyen0623** [1101-GS4719-C/Python-Community-GS4719-C#774] [科技新知]
大家期待嗎！Ｍ1的ipad pro性能相當強大，但最近用起來覺得掉電很快，希望下一代能改善～
https://news.xfastest.com/apple/119076/%E6%9C%80%E5%BC%B7ipad%EF%BC%81-ipad-pro-2022%E6%9C%89%E6%9C%9B%E6%9C%AC%E6%9C%88%E7%99%BC%EF%BC%9Aapple-m2%E5%8A%A0%E6%8C%81/

**Comment by moonnight3503** [comment]
現在ipad越出越厲害，都已經可以取代一台筆電了。

**Reply by chia-kai-chang** [reply]
我目前都用iPad Air 5的LumaFusion軟體剪接影片，然後輸出，影片處理的速度比Windows還快。常用的筆記軟體則是GoodNotes

題外話，10/10國慶日去好事多買了巧控鍵盤，實際使用上覺得好厚重喔~ 有點後悔購買~ 應該會退貨吧~ 

![image](https://user-images.githubusercontent.com/1768891/195396386-a8fb67cc-8c19-46d3-b83e-02d2d63dcca1.png)

**Reply by williamyen0623** [reply]
價格很magic的magic keyboard~ 其實大部分族群用平板蠻少會用到鍵盤的，如果真要用也有價格實惠的logi可以選，我自己就是用logi的k380連ipad，logi也有推出新的鍵盤籌資計畫，鍵盤名稱叫做Rugged Combo 3，早鳥有優惠價，可以去參考看看！

**Reply by chia-kai-chang** [reply]
logi k380我也有，真的好用，打字的感覺很舒服，可以配對三個裝置，價格便宜，放在包包裡面也很好攜帶，值得推薦。

**Comment by ghost** [comment]
強歸強，買不買的起又是另一回事

**Comment by Yunhans** [comment]
我的ipad pro還是用2018年的，到現在還是很夠用，電池也還能撐很久

**Comment by Pietra1226** [comment]
我去年買M1 iPad Pro，確實覺得電量掉很快，不知道是不是太操了。希望M2有改善很多

**Comment by danwan1027** [comment]
iPad 越來越方便，許多單純的文書工作也漸漸可以用 iPad 來取代了

**Comment by a45320614** [comment]
新的ipad pro效能越來越強了，不過對於我這種不需要剪輯高畫質影片及玩遊戲的人來說，還是乖乖買Air就夠用了~

**Comment by Zoe06060606** [comment]
蘋果的晶片是真的歷害 對我而言M1已經很夠用了 但還是很期待他們的新品

**Comment by Kotori0501** [comment]
ipad真的對大學生來說非常實用，但想要求效能好的話，價錢是真的買不起。

**Comment by wangtyphoon** [comment]
apple 究竟能不能讓ARM 架構在X86外殺出一條血路呢?

**Comment by evanchou1204** [comment]
讓我想到iPad 10的詭異規格，這會不會是Apple的策略，出iPhone 14讓大家回去搶iPhone 13 系列，然後出iPad 10代，讓大家回去搶iPad 9，真是太環保了♻️！

**Reply by chia-kai-chang** [reply]
推敲apple的策略很有趣，很多youtuber也用這主題做一系列影片，大家有興趣也可以嘗試把自己的個人觀點製作程影片，向外界發聲

**Comment by Zoe06060606** [comment]
iPad的用途越來越廣汎 就連上課都需要用到 真的是個必需品

**Reply by chia-kai-chang** [reply]
真的，我都用ipad作筆記，自動批改期中考考卷，很棒棒的裝置

**Comment by StevenWanglolz** [comment]
apple真的很搞，本來等著要買ipad10，聽說會變成type c然後支援apple pencil 2，結果只支援pencil 1，而且前幾代的都漲價了 :(

**Comment by Evan0979** [comment]
雖然貴,但apple的產品的品質真的都不錯

**Comment by Floriaand** [comment]
覺得Apple產品出的越來越頻繁

**Comment by CPCpyt** [comment]
希望可以趕快換新的，非常期待新一代！
期待在功能上能夠用創新的改變。
